website/script/visualizer.js

```diff
@@ -288,11 +288,14 @@ function showGame(game, $container, maxWidth, maxHeight, showmovement, isminimal
                 for(var b = 0; b < game.width; b++) {
                     var site = game.frames[frame][Math.floor(loc / game.width)][loc % game.width];
                     if(site.strength == 255) mapGraphics.lineStyle(1, '0xfffff0');
-                    mapGraphics.beginFill(game.players[site.owner].color);
-                    var pw = rw * Math.sqrt(site.strength / 255) / 2, ph = rh * Math.sqrt(site.strength / 255) / 2;
-                    var move = t > 0 ? game.moves[frame][Math.floor(loc / game.width)][loc % game.width] : 0;
+                    if(site.strength != 0) mapGraphics.beginFill(game.players[site.owner].color);
+                    var pw = rw * Math.sqrt(site.strength > 0 ? site.strength / 255 : 0.1) / 2
+                    var ph = rh * Math.sqrt(site.strength > 0 ? site.strength / 255 : 0.1) / 2;
+                    var direction = frame < game.moves.length ? game.moves[frame][Math.floor(loc / game.width)][loc % game.width] : 0;
```

Comment:
Hmm, good question. I think it should, but if you look at the current code in use for visualizer.js at [line 293](https://github.com/HaliteChallenge/Halite/blob/master/website/script/visualizer.js#L293) it uses width for both. This will break on a non-square map though.